tests/run-make/extern-flag-disambiguates/rmake.rs

```diff
@@ -0,0 +1,30 @@
+//@ ignore-cross-compile
+
+use run_make_support::{cwd, run, run_in_tmpdir, rustc};
+
+// Attempt to build this dependency tree:
+//
+//	A.1   A.2
+//	 |\    |
+//	 | \   |
+//        B  \  C
+//         \ | /
+//          \|/
+//           D
```

Comment:
Question: Does A.1 and A.2 need to be aligned with B and C here?

Original Makefile test:

```
#	    A.1   A.2
#	     |\    |
#	     | \   |
#        B  \  C
#         \ | /
#          \|/
#           D
```